### ISSUE TITLE
Kite don't show variables in JS

### ISSUE THREAD

**Issue by alonso804** [kiteco/vim-plugin#345]
Kite don't recommend my variables in Javascript, but it show me in python files.

Javascript file:
![js](https://user-images.githubusercontent.com/59411095/105840963-df8f0e80-5fa1-11eb-9972-d796cbcc9d85.png)

Python file:
![py](https://user-images.githubusercontent.com/59411095/105840964-e027a500-5fa1-11eb-8d55-b46ff67aeb24.png)

**Comment by airblade** [comment]
Try `:let g:kite_supported_languages = ['python', 'javascript']`.

**Comment by alonso804** [comment]
Not working. I have this in my .vimrc

```vim
  autocmd FileType python let b:coc_suggest_disable=1
  autocmd FileType javascript let b:coc_suggest_disable=1
  autocmd FileType scss setl iskeyword+=@-@
  
  let g:kite_supported_languages = ['javascript', 'python']
  
  set completeopt+=menuone
  set completeopt+=noselect
  set completeopt-=preview
  
  set belloff+=ctrlg 
```

**Comment by airblade** [comment]
That's weird.  What does `:echo kite#utils#plugin_version()` say?

**Comment by alonso804** [comment]
1.0.84

**Comment by airblade** [comment]
OK, thanks.  That's up to date.

I assume you have autocomplete on?  `:echo g:kite_auto_complete`

**Comment by airblade** [comment]
Also, does it work when you invoke completion manually with ctrl-x ctrl-u?

**Comment by alonso804** [comment]
No, no work. I'm using CoC, maybe this is the problem. But I have this lines:

```vim
autocmd FileType python let b:coc_suggest_disable=1
autocmd FileType javascript let b:coc_suggest_disable=1
autocmd FileType scss setl iskeyword+=@-@
```

**Comment by airblade** [comment]
I don't yet know why it works for you with Python but not JavaScript.

In the meantime, you could try plugging Kite into [CoC via LSP](https://github.com/kiteco/vim-plugin/blob/master/LSP.md#cocnvim).
